### DISCUSSION TITLE
where is the source code for https://cropperblazor.github.io/demo located ?

### DISCUSSION THREAD

**Discussion by kwoeste11** [CropperBlazor/Cropper.Blazor#301] [General]
In trying to use this library, I came across the demo app located here https://cropperblazor.github.io/demo.  I can't seem to find the source for it. I pulled the repo and don't find it there.  This source behind this app would go a long ways to helping.

**Comment by MaxymGorn** [comment]
@kwoeste11 https://github.com/CropperBlazor/Cropper.Blazor/blob/dev/src/Cropper.Blazor/Client/Pages/CropperDemo.razor, https://github.com/CropperBlazor/Cropper.Blazor/blob/dev/src/Cropper.Blazor/Client/Pages/CropperDemo.razor.cs

**Reply by kwoeste11** [reply]
Thank You.  That should help a lot.
